apps/studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -30,7 +30,10 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
 
   const isFreePlan = subscription?.plan?.id === 'free'
 
-  const { data: membersExceededLimit } = useFreeProjectLimitCheckQuery({ slug: orgSlug })
+  const { data: membersExceededLimit } = useFreeProjectLimitCheckQuery(
+    { slug: orgSlug },
+    { enabled: subscription && subscription.plan.id === 'free' }
```

Comment:
```suggestion
    { enabled: isFreePlan }
```

You literally have this check above :P

Reply:
![](https://media.giphy.com/media/K0Hy2NwI8IXZK/giphy.gif)